index.js

```diff
@@ -106,25 +139,32 @@ function myMap(array, callback) {
 
 // ***** Callbacks - Question 1 *****
 
+function triple(number) {
+  console.log(number * 3)
```

Comment:
This also should be a `return` instead of a `console.log` - the purpose of the `myMap` function is to create a new array with each element based on the callback function, so that callback needs to return a value to be added to the new array. With this code, we'd get back an array of `[undefined, undefined, undefined...]` instead of `[3,6,9...]`